src/test/java/com/znsio/sample/e2e/screen/amazonsearch/AmazonHomeScreen.java

```diff
@@ -0,0 +1,41 @@
+package com.znsio.sample.e2e.screen.amazonsearch;
+
+import com.znsio.e2e.entities.Platform;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.android.amazonsearch.AmazonHomeScreenAndroid;
+import com.znsio.sample.e2e.screen.web.amazonsearch.AmazonHomeScreenWeb;
+import org.apache.commons.lang3.NotImplementedException;
+import org.apache.log4j.Logger;
+
+import static com.znsio.e2e.runner.Runner.fetchDriver;
+import static com.znsio.e2e.runner.Runner.fetchEyes;
+
+public abstract class AmazonHomeScreen{
```

Comment:
it is kind of name redundency like amazonSearch/amazonHomeScreen, remove the additional